### DISCUSSION TITLE
Add API support for merge queues

### DISCUSSION THREAD

**Discussion by ZackKanter** [community/community#56413] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

It would be great if [merge queue](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue) configuration was enabled in the API so that support could be added to [github/safe-settings](https://github.com/github/safe-settings).

**Comment by morremeyer** [comment]
This duplicates #50893
